app/src/main/java/com/lastaoutdoor/lasta/data/auth/AuthRepositoryImpl.kt

```diff
@@ -92,4 +92,20 @@ constructor(
       emit(Response.Failure(e))
     }
   }
+
+  override suspend fun deleteAccount(): Flow<Response<Boolean>> {
+    return try {
+      val userToDelete = auth.currentUser
```

Comment:
You are using a flow as outuput. You should emit loading at the start of the function call.

Reply:
Of course, well done seeing that!